spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReadProjection.java

```diff
@@ -110,10 +110,10 @@ public static void stopSpark() {
   @Override
   protected Record writeAndRead(String desc, Schema writeSchema, Schema readSchema, Record record)
       throws IOException {
-    File parent = temp.newFolder(desc);
+    File parent = new File(temp.toFile(), desc);
```

Comment:
should this use .resolve like in the other classes?

Reply:
https://github.com/apache/iceberg/pull/9380#discussion_r1446433055